### ISSUE TITLE
Php injection seems broken

### ISSUE THREAD

**Issue by AlexandreBonaventure** [bajrangCoder/zed-laravel-blade#5]
Hi, 
I tried this extension but it seems that the php injection is broken. When switching the file to Blade, I get the syntax highlights for html directives, but the php code block above is broken: no highlights, no intellisense.
It would be so nice if this is working, I can help but wanted to first see with you if that's expected. 
Thanks

Tested on Zed v0.149.5

<img width="898" alt="Capture d’écran, le 2024-08-23 à 12 08 20" src="https://github.com/user-attachments/assets/72d2559d-b6bb-48a5-99bb-08d9e2307311">
<img width="892" alt="Capture d’écran, le 2024-08-23 à 12 08 52" src="https://github.com/user-attachments/assets/421fcb6e-39ba-409f-a531-b1c1e2495d50">

**Comment by bajrangCoder** [comment]
It was working fine before (except for the IntelliSense), but it seems the recent Zed updates caused it to break. I'm currently working on a fix.

Thanks for reporting this!

**Comment by bajrangCoder** [comment]
after fixes:
![Screenshot_20240825_092436](https://github.com/user-attachments/assets/2138eb77-e11c-4b8a-8486-63c0e8b4f2e2)

still inline php highlighting isn't working

**Comment by AgungBahtiarr** [comment]
> after fixes: ![Screenshot_20240825_092436](https://private-user-images.githubusercontent.com/71929976/361185308-2138eb77-e11c-4b8a-8486-63c0e8b4f2e2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUyMjk3NDMsIm5iZiI6MTcyNTIyOTQ0MywicGF0aCI6Ii83MTkyOTk3Ni8zNjExODUzMDgtMjEzOGViNzctZTExYy00YjhhLTg0ODYtNjNjMGU4YjRmMmUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAxVDIyMjQwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk4NTAxMjAyMDE5MjVhYTQ0MTA0ZmE5Yzc2ZDEwMGVlNGI1MTA0ZWFkODJjYTVkMTkzNTEyOWFiOTQ1YjQ5MWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.OxZ0L1csC8TiJwCvWTQ82Pf7HlVrEy72atdcNpqQNzY)
> 
> still inline php highlighting isn't working

is the autocomplete for the .blade.php file still not working?

**Comment by bajrangCoder** [comment]
> > after fixes: ![Screenshot_20240825_092436](https://private-user-images.githubusercontent.com/71929976/361185308-2138eb77-e11c-4b8a-8486-63c0e8b4f2e2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUyMjk3NDMsIm5iZiI6MTcyNTIyOTQ0MywicGF0aCI6Ii83MTkyOTk3Ni8zNjExODUzMDgtMjEzOGViNzctZTExYy00YjhhLTg0ODYtNjNjMGU4YjRmMmUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAxVDIyMjQwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk4NTAxMjAyMDE5MjVhYTQ0MTA0ZmE5Yzc2ZDEwMGVlNGI1MTA0ZWFkODJjYTVkMTkzNTEyOWFiOTQ1YjQ5MWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.OxZ0L1csC8TiJwCvWTQ82Pf7HlVrEy72atdcNpqQNzY)
> > still inline php highlighting isn't working
> 
> is the autocomplete for the .blade.php file still not working?

Yes it's not working but it's defined https://github.com/bajrangCoder/zed-laravel-blade/blob/7aba92bc95c50890ba2d4eb1e2f5aeb2e795fce2/languages/blade/config.toml#L19

**Comment by bajrangCoder** [comment]
These things are now working in v0.1.0

**Comment by AlexandreBonaventure** [comment]
Thanks a lot! Eager to try, but I think you guys did not publish the extension yet, it's still on 0.0.5 in the extensions store

<img width="447" alt="Image" src="https://github.com/user-attachments/assets/25eabcb3-1a10-4e76-b2f4-459a9df10421" />

**Comment by bajrangCoder** [comment]
> Thanks a lot! Eager to try, but I think you guys did not publish the extension yet, it's still on 0.0.5 in the extensions store
> 
> <img alt="Image" width="447" src="https://private-user-images.githubusercontent.com/4596409/434431242-25eabcb3-1a10-4e76-b2f4-459a9df10421.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQ4NTQ0NTIsIm5iZiI6MTc0NDg1NDE1MiwicGF0aCI6Ii80NTk2NDA5LzQzNDQzMTI0Mi0yNWVhYmNiMy0xYTEwLTRlNzYtYjJmNC00NTlhOWRmMTA0MjEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDQxNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA0MTdUMDE0MjMyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MDAzODY4MmM1MWZmMzk1ZTk0YWM1ZGIxMzY0NTljNGM5NDQ5NGNhZDQ5YjkxMmE3YjQxYmZjMTUzZGRhNTlhNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.TC7SHjojmBWIQJePPeD8qJUEyfUO0vuhMqAjsl2wcmk">

Actually the pr wasn't merged but it is merged

**Comment by bajrangCoder** [comment]
Now it's merged and also update is available on zed
